Skate_Behaviour_IO.m

```diff
@@ -114,40 +137,45 @@
                 fprintf('Next marker: ')
                 fprintf(num2str(next_marker))
                 fprintf(' , ')
+                
+                
             end
             
-            if this_marker == nTarget
+            if this_marker == nTarget || this_marker == pTarget
```

Comment:
again not sure the difference between p and n

Reply:
as mentioned before, n is for the original set of triggers, and p is to identify the new trigger people.